exception-handling/NullReferenceException/NullReferenceException/ExceptionMethods.cs

```diff
@@ -0,0 +1,129 @@
+﻿namespace NullReference
+{
+    public class ExceptionMethods
+    {
+        public List<string> StudentList() 
+        {
+            List<string> studentList = null;
+
+            studentList.Add("John Doe");
+
+            return studentList;
+        }
+
+        public List<string> FixedStudentList()
+        {
+            List<string> studentList = new List<string>();
+
+            studentList.Add("John Doe");
+
+            return studentList;
+        }
+
+        public int[] RandomNumbers()
+        {
+            var rand = new Random();
+
+            int[] numbers = null;
+
+            for(int i = 0; i < numbers.Length; i++)
+                numbers[i] = rand.Next();
```

Comment:
Can we standardize the use of brackets for loops? There is a couple of methods that use brackets in loops and most that don't. It doesn't bother me which we choose, but best to be consistent.